README.md

```diff
@@ -62,6 +62,13 @@ By default, balenaSound bluetooth will connect using Secure Simple Pairing mode.
 
 **Note**: Legacy Mode is no longer allowed on [iOS](https://developer.apple.com/accessories/Accessory-Design-Guidelines.pdf) devices.
 
+### Bluetooth Scripts
+
+By default, balenaSound will run no scripts at connect and disconnect of a bluetooth device. If you would like to activate this, set the `BLUETOOTH_SCRIPTS` environment variable to `true`.
+There are two python scripts in the `./bluetooth-audio/bluetooh-scripts/` directory, which run on connect and disconnect. If you would like to control over GPIO a external amplifier can you use gpiozero which is preinstalled. 
```

Comment:
In here I would rephrase it so that we get across this two points:
- Sample scripts are provided in `./bluetooth-audio/bluetooth-scripts/` directory
- You should/can edit them to your linking